docs/source/conf.py

```diff
@@ -186,9 +186,10 @@ def setup(app):
     app.connect("autodoc-skip-member", skip)
 """
 
-# awful hack to get rtd builder to install pytorch
-os.system('pip install awscli')
-os.system('aws s3 --no-sign-request sync s3://pyro-ppl/ci tmp --exclude "*" --include "*-cp27-*";')
-os.system('pip install tmp/*')
-os.system('rm -r tmp')
-os.system('pip install -e .')
+# @jpchen's hack to get rtd builder to install pytorch
```

Comment:
thats what `git blame` is for :P

Reply:
I always find it useful when modifying config files to add a comment on who added a section, especially if it was I who modified the section. That way later on I can see how my config differs from the original. While `git blame` can do some of this, it can easily get lost due to a file move.